drivers/hmis/app/graphql/types/hmis_schema/ce_referral_step.rb

```diff
@@ -31,5 +31,9 @@ def workflow_node
     def swimlane
       load_ar_association(object, :swimlane)&.name
     end
+
+    def form_definition
+      load_ar_association(object.node, :form_definition)
```

Comment:
is there a reason not to use data loader to load association `object.node`? this is a little funny to me because it will be N+1 anyway when loading the node, I think.

Reply:
Hrm! I made this improvement, but there must still be something I'm not understanding either in the dataloader or how I'm setting up [this spec test](https://github.com/greenriver/hmis-warehouse/pull/5204/files#diff-2dd19f7adb4deb398e998a7ecfb0e2214ba3abf99c74b883af00cc77edbd8d89R201-R233). After running that test with both versions of the code, I didn't see a change in the # of db queries made.